pyuvdata/uvdata/uvdata.py

```diff
@@ -770,6 +771,15 @@ def __init__(self):
 
         super(UVData, self).__init__()
 
+    @staticmethod
+    def new(**kwargs):
+        """Initialize a new UVData object from keyword arguments.
+
+        All parameters passed through to the
+        :func:`~pyuvdata.uvdata.initializers.new_uvdata` function.
+        """
```

Comment:
We need to decide how we're going to build the docs for this method. If we leave it like this, we need to add the uvdata initializers module to the docs. But I'd favor not doing that and instead properly documenting the keywords here so they're picked up on the object docs.

Reply:
I took an in-between approach. I left the literal docstring in the function, and just attach that docstring to the `.new()` method. It should show up in docs well.

Reply:
This is what I am seeing in the RTD build at the moment:

<img width="322" alt="Screen Shot 2023-04-20 at 2 15 03 PM" src="https://user-images.githubusercontent.com/3084869/233489217-d93cdaf5-cb32-4036-90b2-cf875b9b4be2.png">

Reply:
Looks like your new approach worked. Thanks for fixing it!